src/main/java/edu/ie3/util/TimeUtil.java

```diff
@@ -59,6 +66,10 @@ public String toString(ZonedDateTime zonedDateTime) {
     return dateTimeFormatter.format(zonedDateTime);
   }
 
+  public String toLocalDateTimeString(ZonedDateTime zonedDateTime) {
```

Comment:
static?

Reply:
Correct me if I am wrong but: Since this method only returns a String with the given pattern there is no locale to be considered.

Reply:
@t-ober but given a zoned datetime that could have a different timezone, wouldn't you want the local datetime to be converted to the desired timezone before "discarding" timezone info?